ors-engine/src/main/java/org/heigit/ors/routing/configuration/RouteProfileConfiguration.java

```diff
@@ -64,7 +64,7 @@ public class RouteProfileConfiguration {
 
     private int maximumVisitedNodesPT = 1000000;
 
-    private boolean turnCostEnabled = false;
+    private boolean turnCostEnabled = false;//FIXME: even though the field is read by external methods, its setter is never called.
```

Comment:
Shouldn't be the FIXME in code instead of the docs?

Reply:
It is, this appears to be an actual smell in the code @aoles came across while writing the docs (I'm guessing), that's why the comment got mixed in here with the docs commits. I personally think we shouldn't get too finicky about the commit/PR semantics...

Reply:
That's correct, I figured out we might want to change it. Obviously addressing the TODO is outside of the PR's scope .